config/import-control.xml

```diff
@@ -7,7 +7,18 @@
     <allow pkg="java.io"/>
     <allow pkg="java.util"/>
     <allow pkg="com.github.checkstyle.regression.data"/>
+    <allow pkg="com.github.checkstyle.regression.util"/>
+    <allow pkg="org.immutables"/>
```

Comment:
@Luolc I asked romani his opinion on this, and he isn't a fan of "magic" builders either.
Since this isn't the main repository, we could keep this here if we really wanted to.

My only concern right now is code coverage as `ModuleInfo` isn't showing any, even for `hasCheckerParent` which is a branch because of the equality test.